crypto/rc4/asm/rc4-x86_64.pl

```diff
@@ -552,7 +552,6 @@ sub RC4_loop {
 .asciz	"rc4(8x,int)"
 .asciz	"rc4(8x,char)"
 .asciz	"rc4(16x,int)"
```

Comment:
I think these RC4_options() and other FOO_options() would be good candidates for removal. They're only used in the 'app' (version command for BN_options, for example).

Reply:
I'll make a separate PR for those.  Looks like AES, BF, BN, rc4 among others.